R/MetAlyzer-seclass.R

```diff
@@ -152,70 +151,10 @@ getQuantStatus <- function(metalyzer) {
 #'
 #' aggregated_data <- getAggregatedData(metalyzer)
 getAggregatedData <- function(metalyzer) {
-  aggregated_data <- metalyzer@aggregated_data
+  aggregated_data <- metadata(metalyzer)$aggregated_data
   return(aggregated_data)
 }
 
-# === Display MetAlyzer class ===
-
-#' @title Show a 'MetAlyzer' object
-#'
-#' @description This function shows a summary of the 'MetAlyzer' slot values.
-#'
-#' @param metalyzer MetAlyzer object
-#' @return A summary of the MetAlyzer object
-#' @importFrom methods show
-#' @export
-#'
-#' @examples
-#' metalyzer <- MetAlyzer_dataset(file_path = extraction_data())
-#'
-#' show(metalyzer)
-#' # or
-#' metalyzer
-setMethod("show",
```

Comment:
Is there an equivalent function available for SE objects or why is this function removed?